core/src/main/java/org/infinispan/configuration/cache/Configuration.java

```diff
@@ -150,27 +150,18 @@ public CompatibilityModeConfiguration compatibility() {
 
    @Override
    public String toString() {
-      return "Configuration{" +
-            "clustering=" + clusteringConfiguration +
-            ", customInterceptors=" + customInterceptorsConfiguration +
-            ", dataContainer=" + dataContainerConfiguration +
-            ", deadlockDetection=" + deadlockDetectionConfiguration +
-            ", eviction=" + evictionConfiguration +
-            ", expiration=" + expirationConfiguration +
-            ", indexing=" + indexingConfiguration +
-            ", invocationBatching=" + invocationBatchingConfiguration +
-            ", jmxStatistics=" + jmxStatisticsConfiguration +
-            ", persistence=" + persistenceConfiguration +
-            ", locking=" + lockingConfiguration +
-            ", modules=" + moduleConfiguration +
-            ", security=" + securityConfiguration +
-            ", storeAsBinary=" + storeAsBinaryConfiguration +
-            ", transaction=" + transactionConfiguration +
-            ", versioning=" + versioningConfiguration +
-            ", unsafe=" + unsafeConfiguration +
-            ", sites=" + sitesConfiguration +
-            ", compatibility=" + compatibilityConfiguration +
-            '}';
+      return "Configuration [clusteringConfiguration=" + clusteringConfiguration + ", customInterceptorsConfiguration="
```

Comment:
Is this just a formatting change? :)

The PR is already big enough to add cosmetic changes into it :)